driver/src/orderbook/mod.rs

```diff
@@ -39,7 +39,7 @@ pub trait StableXOrderBookReading {
 }
 
 /// The different kinds of orderbook readers.
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
This can probably be `Copy` as well so you don't need an explicit call to `Clone`
```suggestion
#[derive(Copy, Clone, Debug)]
```

Reply:
I'll give it a try.

Reply:
You mean on `OrderbookReaderKind`? Should I also be removing the clone? What is this? I don't even explicitly clone this anywhere.